tests/_utils/run_services

```diff
@@ -27,6 +27,7 @@ TIKV_ADDR="127.0.0.1:2016"
 TIKV_STATUS_ADDR="127.0.0.1:2018"
 TIKV_COUNT=3
 TIFLASH_STATUS="127.0.0.1:17000"
```

Comment:
remove this line?

Reply:
`$TIFLASH_STATUS` would still be used at make_tiflash_config:

https://github.com/pingcap/br/blob/56e151ad8b672272772cd2e37c6c26b148d9942e/tests/_utils/make_tiflash_config#L16

I think maybe leave it here would be better?

Reply:
Ok.